1_fetch/src/fetch_river_geometry.R

```diff
@@ -1,16 +1,17 @@
-fetch_river_geoms <- function(ind_file, view_polygon, sites, streamorder) {
+fetch_river_geoms <- function(ind_file, view_polygon, sites_ind) {
 
-  sites <- readRDS(sc_retrieve(sites))
+  sites <- readRDS(sc_retrieve(sites_ind))
 
   if(nrow(sites) > 30) {
     set.seed(42)
     sites <- sites[sample(seq_len(nrow(sites)), 30),]
   }
 
-  bbox <- sf::st_bbox(sf::st_transform(view_polygon, 4326))
+  bbox <- st_bbox(st_transform(view_polygon, 4326))
 
   postURL <- "https://cida.usgs.gov/nwc/geoserver/nhdplus/ows"
 
+  streamorder <- 4 # getting more than we need but not all!
```

Comment:
Do we want to put options like this in the viz_config.yml? I'm thinking of the case where a storm covers a smaller geographic area and maybe we want to have the same density of streams in the viewing area, but this will be dependent on the stream order.

Reply:
I suppose this is just major rivers and we'd still grab smaller streams with the site list

Reply:
I want to change the handling of this. It is a pre-filter so we don't download way too much data, the later filters on drainage area go to more like streamorder 5 or 6. There's a chance it could be removed as downloading a ton of data isn't really a blocker. 

Note the difference between the upstream extent of the light and dark blue lines. That is a result of this filter only grabbing 4th order and higher streams for the dark blue lines. If I switch this to download all major rivers, we would be able to extend the major rivers all the way up stream if we wanted to.